cannot install cargo-scaffold v0.8.5 on mac intel

When trying to install cargo-scaffold, I got the following error:
```
cargo install cargo-scaffold
    Updating crates.io index
  Installing cargo-scaffold v0.8.5
   Compiling libc v0.2.135
   Compiling proc-macro2 v1.0.47
   Compiling quote v1.0.21
   Compiling unicode-ident v1.0.5
   Compiling syn v1.0.102
   Compiling memchr v2.5.0
   Compiling autocfg v1.1.0
   Compiling thiserror v1.0.37
   Compiling pkg-config v0.3.25
   Compiling version_check v0.9.4
   Compiling cfg-if v1.0.0
   Compiling lazy_static v1.4.0
   Compiling regex-syntax v0.6.27
   Compiling core-foundation-sys v0.8.3
   Compiling serde_derive v1.0.145
   Compiling unicode-width v0.1.10
   Compiling bitflags v1.3.2
   Compiling tinyvec_macros v0.1.0
   Compiling serde v1.0.145
   Compiling log v0.4.17
   Compiling ucd-trie v0.1.5
   Compiling fastrand v1.8.0
   Compiling unicode-bidi v0.3.8
   Compiling percent-encoding v2.2.0
   Compiling remove_dir_all v0.5.3
   Compiling itoa v1.0.4
   Compiling once_cell v1.15.0
   Compiling crc32fast v1.3.2
   Compiling adler v1.0.2
   Compiling fnv v1.0.7
   Compiling serde_json v1.0.86
   Compiling ryu v1.0.11
   Compiling hashbrown v0.12.3
   Compiling native-tls v0.2.10
   Compiling bytes v1.2.1
   Compiling unicode-segmentation v1.10.0
   Compiling linked-hash-map v0.5.6
   Compiling vec_map v0.8.2
   Compiling anyhow v1.0.65
   Compiling strsim v0.8.0
   Compiling wildmatch v1.1.0
   Compiling ansi_term v0.12.1
   Compiling number_prefix v0.3.0
   Compiling same-file v1.0.6
   Compiling md5 v0.7.0
   Compiling shell-words v1.1.0
   Compiling proc-macro-error-attr v1.0.4
   Compiling proc-macro-error v1.0.4
   Compiling indexmap v1.9.1
   Compiling textwrap v0.11.0
   Compiling tinyvec v1.6.0
   Compiling form_urlencoded v1.1.0
   Compiling miniz_oxide v0.5.4
   Compiling http v0.2.8
   Compiling yaml-rust v0.4.5
   Compiling heck v0.3.3
   Compiling walkdir v2.3.2
   Compiling unicode-normalization v0.1.22
   Compiling aho-corasick v0.7.19
   Compiling bstr v0.2.17
   Compiling idna v0.3.0
   Compiling flate2 v1.0.24
   Compiling terminal_size v0.1.17
   Compiling tempfile v3.3.0
   Compiling security-framework-sys v2.6.1
   Compiling core-foundation v0.9.3
   Compiling termios v0.3.3
   Compiling atty v0.2.14
   Compiling jobserver v0.1.25
   Compiling regex v1.6.0
   Compiling console v0.15.2
   Compiling clap v2.34.0
   Compiling url v2.3.1
   Compiling security-framework v2.7.0
   Compiling cc v1.0.73
   Compiling Inflector v0.11.4
   Compiling console v0.11.3
   Compiling globset v0.4.9
   Compiling console v0.12.0
   Compiling indicatif v0.15.0
   Compiling openssl-sys v0.9.76
   Compiling libz-sys v1.1.8
   Compiling libssh2-sys v0.2.23
   Compiling jsonnet-sys v0.17.0
   Compiling libgit2-sys v0.13.4+1.4.2
   Compiling str_inflector v0.12.0
   Compiling dialoguer v0.6.2
   Compiling thiserror-impl v1.0.37
   Compiling attohttpc v0.17.0
   Compiling try_match_inner v0.4.0
   Compiling structopt-derive v0.4.18
   Compiling try_match v0.3.0
   Compiling pest v2.4.0
   Compiling enquote v1.1.0
   Compiling buildstructor v0.3.2
   Compiling pest_meta v2.4.0
   Compiling jsonnet-rs v0.17.0
   Compiling structopt v0.3.26
   Compiling pest_generator v2.4.0
   Compiling pest_derive v2.4.0
   Compiling git2 v0.14.4
   Compiling toml v0.5.9
   Compiling serde_yaml v0.8.26
   Compiling handlebars v4.3.5
   Compiling handlebars_misc_helpers v0.12.1
   Compiling cargo-scaffold v0.8.5
error: linking with `clang` failed: exit status: 1
  |
  = note: "clang" "-m64" "-arch" "x86_64" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/rustchFOBtI/symbols.o" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/cargo_scaffold-016dc3176c6ac5d4.cargo_scaffold.7db91716-cgu.0.rcgu.o" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/cargo_scaffold-016dc3176c6ac5d4.cargo_scaffold.7db91716-cgu.1.rcgu.o" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/cargo_scaffold-016dc3176c6ac5d4.cargo_scaffold.7db91716-cgu.10.rcgu.o" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/cargo_scaffold-016dc3176c6ac5d4.cargo_scaffold.7db91716-cgu.11.rcgu.o" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/cargo_scaffold-016dc3176c6ac5d4.cargo_scaffold.7db91716-cgu.12.rcgu.o" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/cargo_scaffold-016dc3176c6ac5d4.cargo_scaffold.7db91716-cgu.13.rcgu.o" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/cargo_scaffold-016dc3176c6ac5d4.cargo_scaffold.7db91716-cgu.14.rcgu.o" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/cargo_scaffold-016dc3176c6ac5d4.cargo_scaffold.7db91716-cgu.15.rcgu.o" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/cargo_scaffold-016dc3176c6ac5d4.cargo_scaffold.7db91716-cgu.2.rcgu.o" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/cargo_scaffold-016dc3176c6ac5d4.cargo_scaffold.7db91716-cgu.3.rcgu.o" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/cargo_scaffold-016dc3176c6ac5d4.cargo_scaffold.7db91716-cgu.4.rcgu.o" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/cargo_scaffold-016dc3176c6ac5d4.cargo_scaffold.7db91716-cgu.5.rcgu.o" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/cargo_scaffold-016dc3176c6ac5d4.cargo_scaffold.7db91716-cgu.6.rcgu.o" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/cargo_scaffold-016dc3176c6ac5d4.cargo_scaffold.7db91716-cgu.7.rcgu.o" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/cargo_scaffold-016dc3176c6ac5d4.cargo_scaffold.7db91716-cgu.8.rcgu.o" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/cargo_scaffold-016dc3176c6ac5d4.cargo_scaffold.7db91716-cgu.9.rcgu.o" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/cargo_scaffold-016dc3176c6ac5d4.3lmhlazx4v4373a1.rcgu.o" "-L" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps" "-L" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/build/libgit2-sys-e61b018a399c0e74/out/build" "-L" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/build/libssh2-sys-5fe11a225dfdf310/out/build" "-L" "/usr/local/opt/openssl@1.1/lib" "-L" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/build/jsonnet-sys-792914db1326ea36/out" "-L" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libcargo_scaffold-e38084eb270a5b8f.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libshell_words-1515c30f51491ae0.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libhandlebars_misc_helpers-f5745e7cb61abd39.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libenquote-da42eca7e4722e0a.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libserde_yaml-33058dbf94eaedc0.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libyaml_rust-32601fa5506f8973.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/liblinked_hash_map-0971fc25a868eae7.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libinflector-d9de973fab6fb225.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libjsonnet-0252661b06a0b2e8.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libjsonnet_sys-5eeda51008f91631.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libattohttpc-e7ad4474ea04d853.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libnative_tls-eb7c44a55a9db0b4.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libsecurity_framework-53c16223f3f906b2.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libsecurity_framework_sys-3fa65ddf4e3b8755.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libcore_foundation-f20ee6c73f507386.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libcore_foundation_sys-0ca7de4c5d16d2a1.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libwildmatch-8ca3f3f70c766426.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libflate2-508a2ddf3b431ee1.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libminiz_oxide-ecdb40c2cdc5ef2f.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libadler-ecf5b08bac00a973.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libcrc32fast-6c02979cb24654ae.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libhttp-1926b76f9aabc819.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libbytes-49c4f9b6fe48ec62.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libmd5-fc0ed7b13e284a82.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libtoml-3bff894f240ef575.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libindexmap-a940542334eb1b62.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libhashbrown-5e7a6f097f2ca45c.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libwalkdir-e1cb1b713b303a42.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libsame_file-b5b3ebbe21070585.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libstructopt-9bc92c33a724e792.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libclap-b707467d5e2b08df.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libvec_map-b7022e9deccad96a.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libtextwrap-5be7511a4ce3e7fe.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libstrsim-ee28a658ae804063.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libatty-f4d246cd2995b05a.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libansi_term-8affd236fd094f7c.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libglobset-a3a4c254242106b3.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libfnv-37777db1b355d5c7.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libbstr-58e026307d3b8695.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libhandlebars-05c70069d857890d.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libpest-8891df1d43a1dddc.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libucd_trie-dbe261391e886447.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libthiserror-805c25b81e1045bb.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libserde_json-55e3799720e7e9d9.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libryu-d1053c5ce0a10b6d.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libitoa-cca5616e26f96c56.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libserde-ab1a4fb57f186ad0.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libgit2-27ef9c7af7015e6d.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/liburl-6dce9ad114a3dc51.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libidna-7bab3763cb1bf97a.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libunicode_normalization-f9e453dc533a4293.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libtinyvec-1912cfa7fc9ecd86.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libtinyvec_macros-a20290cd90cce085.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libunicode_bidi-4703de0cb70e2024.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libform_urlencoded-26cfe6bbc290dc29.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libpercent_encoding-1f4ff2cae73dc27d.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/liblog-c014a7cb9517e035.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/liblibgit2_sys-b981921dafb95433.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/liblibssh2_sys-b0d41bd446293a58.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libopenssl_sys-c64435f1540b8c5f.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/liblibz_sys-c0f7572a5bab6ca5.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libbitflags-7e70a6941a88af1d.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libdialoguer-80fb809bbda7d604.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libtempfile-5a67c5b0ec594a75.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libcfg_if-71ff7bf3845d02bd.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libfastrand-0cb96140797fc87c.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libremove_dir_all-d43bba1e7fc3daca.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libconsole-56ca81436e6cc571.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libconsole-3176a0d67d50d8e7.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libtermios-ec774d7487f70150.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libterminal_size-f3aa42666c493b11.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/liblibc-29d4e95ff1ad37bc.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libregex-fdf824cc146095a8.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libaho_corasick-50efd8dbf0f5288c.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libmemchr-1896e7fd7f622c59.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libregex_syntax-7d53bdc7dfbc7f51.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libunicode_width-6b20537f4fa0a02b.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/liblazy_static-b6807f579041be0f.rlib" "/private/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/libanyhow-e558d952ff5de167.rlib" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libstd-0f7ee384278ce82b.rlib" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libpanic_unwind-6023318e4257fdb3.rlib" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libobject-50ed95d28fda9497.rlib" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libmemchr-114781e2905bc242.rlib" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libaddr2line-841a5df74cbbcf8e.rlib" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libgimli-9b35810dd2e8e276.rlib" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/librustc_demangle-d44decaafa04c51d.rlib" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libstd_detect-aa335e35e4a7724c.rlib" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libhashbrown-7ac72202be300078.rlib" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libminiz_oxide-2930c6f21f36f92f.rlib" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libadler-654445a53da668f3.rlib" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/librustc_std_workspace_alloc-ce034a3eed8d4113.rlib" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libunwind-51412ab8efb0f481.rlib" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libcfg_if-1c20aac4d9e33893.rlib" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/liblibc-5559092a2ede5191.rlib" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/liballoc-05250b6a4768a099.rlib" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/librustc_std_workspace_core-10f98b32877a2067.rlib" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libcore-56d27115b82c9961.rlib" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libcompiler_builtins-8b28a0a374c38504.rlib" "-lc++" "-framework" "Security" "-framework" "CoreFoundation" "-liconv" "-framework" "Security" "-framework" "CoreFoundation" "-lssl" "-lcrypto" "-lz" "-lc" "-liconv" "-lSystem" "-lresolv" "-lc" "-lm" "-liconv" "-L" "/Users/yannsimon/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib" "-o" "/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6/release/deps/cargo_scaffold-016dc3176c6ac5d4" "-Wl,-dead_strip" "-nodefaultlibs"
  = note: Undefined symbols for architecture x86_64:
            "std::__1::basic_stringbuf<char, std::__1::char_traits<char>, std::__1::allocator<char> >::str() const", referenced from:
                jsonnet_evaluate_snippet_aux(JsonnetVm*, char const*, char const*, int*, (anonymous namespace)::EvalKind) in libjsonnet_sys-5eeda51008f91631.rlib(libjsonnet.o)
                StaticError::toString() const in libjsonnet_sys-5eeda51008f91631.rlib(libjsonnet.o)
                lex_number(char const*&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, Location const&) in libjsonnet_sys-5eeda51008f91631.rlib(lexer.o)
                jsonnet_lex(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, char const*) in libjsonnet_sys-5eeda51008f91631.rlib(lexer.o)
                jsonnet_unparse_number(double) in libjsonnet_sys-5eeda51008f91631.rlib(parser.o)
                jsonnet_parse(Allocator*, std::__1::list<Token, std::__1::allocator<Token> >&) in libjsonnet_sys-5eeda51008f91631.rlib(parser.o)
                (anonymous namespace)::Parser::parse(unsigned int) in libjsonnet_sys-5eeda51008f91631.rlib(parser.o)
                ...
            "std::__1::basic_filebuf<char, std::__1::char_traits<char> >::basic_filebuf()", referenced from:
                try_path(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in libjsonnet_sys-5eeda51008f91631.rlib(libjsonnet.o)
            "std::__1::basic_filebuf<char, std::__1::char_traits<char> >::~basic_filebuf()", referenced from:
                try_path(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in libjsonnet_sys-5eeda51008f91631.rlib(libjsonnet.o)
            "std::__1::basic_ifstream<char, std::__1::char_traits<char> >::open(char const*, unsigned int)", referenced from:
                try_path(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in libjsonnet_sys-5eeda51008f91631.rlib(libjsonnet.o)
            "std::__1::basic_stringbuf<char, std::__1::char_traits<char>, std::__1::allocator<char> >::str(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&)", referenced from:
                jsonnet_lex(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, char const*) in libjsonnet_sys-5eeda51008f91631.rlib(lexer.o)
            "VTT for std::__1::basic_ifstream<char, std::__1::char_traits<char> >", referenced from:
                try_path(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in libjsonnet_sys-5eeda51008f91631.rlib(libjsonnet.o)
            "VTT for std::__1::basic_stringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> >", referenced from:
                std::__1::basic_stringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> >::basic_stringstream() in libjsonnet_sys-5eeda51008f91631.rlib(libjsonnet.o)
                jsonnet_evaluate_snippet_aux(JsonnetVm*, char const*, char const*, int*, (anonymous namespace)::EvalKind) in libjsonnet_sys-5eeda51008f91631.rlib(libjsonnet.o)
                StaticError::toString() const in libjsonnet_sys-5eeda51008f91631.rlib(libjsonnet.o)
                lex_number(char const*&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, Location const&) in libjsonnet_sys-5eeda51008f91631.rlib(lexer.o)
                std::__1::basic_stringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> >::basic_stringstream() in libjsonnet_sys-5eeda51008f91631.rlib(lexer.o)
                jsonnet_lex(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, char const*) in libjsonnet_sys-5eeda51008f91631.rlib(lexer.o)
                jsonnet_unparse_number(double) in libjsonnet_sys-5eeda51008f91631.rlib(parser.o)
                ...
            "vtable for std::__1::basic_ifstream<char, std::__1::char_traits<char> >", referenced from:
                try_path(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in libjsonnet_sys-5eeda51008f91631.rlib(libjsonnet.o)
            NOTE: a missing vtable usually means the first non-inline virtual member function has no definition.
            "vtable for std::__1::basic_stringbuf<char, std::__1::char_traits<char>, std::__1::allocator<char> >", referenced from:
                std::__1::basic_stringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> >::basic_stringstream() in libjsonnet_sys-5eeda51008f91631.rlib(libjsonnet.o)
                jsonnet_evaluate_snippet_aux(JsonnetVm*, char const*, char const*, int*, (anonymous namespace)::EvalKind) in libjsonnet_sys-5eeda51008f91631.rlib(libjsonnet.o)
                StaticError::toString() const in libjsonnet_sys-5eeda51008f91631.rlib(libjsonnet.o)
                lex_number(char const*&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, Location const&) in libjsonnet_sys-5eeda51008f91631.rlib(lexer.o)
                std::__1::basic_stringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> >::basic_stringstream() in libjsonnet_sys-5eeda51008f91631.rlib(lexer.o)
                jsonnet_lex(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, char const*) in libjsonnet_sys-5eeda51008f91631.rlib(lexer.o)
                jsonnet_unparse_number(double) in libjsonnet_sys-5eeda51008f91631.rlib(parser.o)
                ...
            NOTE: a missing vtable usually means the first non-inline virtual member function has no definition.
            "vtable for std::__1::basic_stringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> >", referenced from:
                std::__1::basic_stringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> >::basic_stringstream() in libjsonnet_sys-5eeda51008f91631.rlib(libjsonnet.o)
                std::__1::basic_stringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> >::basic_stringstream() in libjsonnet_sys-5eeda51008f91631.rlib(lexer.o)
                std::__1::basic_stringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> >::basic_stringstream() in libjsonnet_sys-5eeda51008f91631.rlib(parser.o)
                std::__1::basic_stringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> >::basic_stringstream() in libjsonnet_sys-5eeda51008f91631.rlib(desugarer.o)
                std::__1::basic_stringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> >::basic_stringstream() in libjsonnet_sys-5eeda51008f91631.rlib(vm.o)
                std::__1::basic_stringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> >::basic_stringstream() in libjsonnet_sys-5eeda51008f91631.rlib(string_utils.o)
            NOTE: a missing vtable usually means the first non-inline virtual member function has no definition.
          ld: symbol(s) not found for architecture x86_64
          clang-12: error: linker command failed with exit code 1 (use -v to see invocation)
          

error: could not compile `cargo-scaffold` due to previous error
error: failed to compile `cargo-scaffold v0.8.5`, intermediate artifacts can be found at `/var/folders/hn/zkgzcdkn49x3xc8ftxf408tr0000gn/T/cargo-installRZkch6`
```
I've tried to search for it, but could not find anything related in other issues.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -19,7 +19,7 @@ walkdir = "2.3.1"
 toml = "0.5.6"
 git2 = "0.14"
 md5 = "0.7.0"
-handlebars_misc_helpers = { version = "0.12", optional = true }
+handlebars_misc_helpers = { version = "0.12", default-features = false, features = ["string", "http_attohttpc", "json"], optional = true }
 indicatif = "0.15.0"
 console = "0.12.0"
 globset = "0.4.5"

--- a/rust-toolchain.toml
+++ b/rust-toolchain.toml
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "1.60.0"
+channel = "1.64.0"

